TutorialApplication/CreateEmployeeViewController.m

```diff
@@ -19,10 +42,14 @@ - (IBAction)addNewEmployee
     NSString *salary = self.salaryTextField.text;
     int salaryInt = [salary intValue];
     
+    NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+    [dateFormatter setDateFormat:@"yyyy-MM-dd"];
+    
     FFEmployee *newEmployee = [NSEntityDescription insertNewObjectForEntityForName:@"FFEmployee" inManagedObjectContext:[DatabaseController sharedInstance].context];
     newEmployee.firstName = firstName;
     newEmployee.lastName = lastName;
     newEmployee.salary = salaryInt;
+    newEmployee.dateOfBirth = [dateFormatter dateFromString:self.dateOfBirthTextField.text];
```

Comment:
Bad approach. You create string from date and then date from string. Why not to store the date in a property?